mesonbuild/mesondata.py

```diff
@@ -282,7 +282,7 @@
 
 macro(meson_ps_disabled_function)
   message(WARNING "The function '${ARGV0}' is disabled in the context of CMake subporjects.\n"
-                  "This should not be an issue but may lead to compilaton errors.")
+                  "This should not be an issue but may lead to compilation errors.")
```

Comment:
Needs to be fixed in the master copy at https://github.com/mesonbuild/meson/blob/e6a167ce092a36017a4ff56b3fb045be62377a25/mesonbuild/cmake/data/preload.cmake#L19

Reply:
This is already fixed, you may have picked the wrong commit.
https://github.com/mesonbuild/meson/blob/57b3ef43ed9c81fa18fa17bb4488a119e632eecf/mesonbuild/cmake/data/preload.cmake#L19

Reply:
Sorry, I mis-analyzed the cause of the CI failure (and I'm not sure why I didn't notice that change in the commit diff).

As the CI notes, you need to edit the master copy of the data file first, then run `./tools/gen_data.py` to update mesondata.py -- but it does more than copy over the file. Checking out your PR, then running the tool, adds the following commit diff on top of your changes:

```diff
diff --git a/mesonbuild/mesondata.py b/mesonbuild/mesondata.py
index b1efcbed3..110103e7b 100644
--- a/mesonbuild/mesondata.py
+++ b/mesonbuild/mesondata.py
@@ -1,4 +1,4 @@
-# Copyright 2020 The Meson development team
+# Copyright 2021 The Meson development team
 
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -388,7 +388,7 @@ mesondata = {
     ),
     'cmake/data/preload.cmake': DataFile(
         Path('cmake/data/preload.cmake'),
-        '2b4e632aeb74acb2b441880cf85c0b6fcab03e75b182d3077715a97e739a7918',
+        '7b90f6a60e3b53ecb3a9a206bcdb0e077b9b28452c54fc6cf1295f627d1dabc7',
         file_3_data_preload_cmake,
     ),
 }
```